src/shared/components/case-viewer/case-viewer.component.spec.ts

```diff
@@ -1,3 +1,4 @@
+import { HttpErrorResponse } from '@angular/common/http';
```

Comment:
I'm seeing an `xdescribe(...)` for all the tests in this suite...?

Reply:
@paul-graham - that's a bit odd... I tried an `fdescribe` and all those tests fail! They ought to be fixed but that would be outside the scope of this PR, so I've left as `xdescribe` (as it was) for now.

Reply:
Okay, cool. So long as you're aware of it. We should create a tech debt ticket to tidy up any tests that have an `x` prefix (`xdescribe` or `xit`) - either fix them or remove them. 😄